prisma/seed.ts

```diff
@@ -39,60 +52,70 @@ const createParks = async () => {
   await prisma.park.createMany({
     data: [
       {
+        id: 1,
```

Comment:
Warum hosch do überall jetzt id's dazua to?

Reply:
weil ma so endlich fixe qr code links generieren kann. ohne ids macht d datenbank halt aufsteigende ids wo sich jedes mol ändern.

Reply:
I würds so macha wie i mol vorgschlaga hob - nämlich das ma a schnittstelle machen welche als query token den namen/kürzel vom Sportplatz ahnimmt und denn uf die sita weiterleitet - des werden mir o irgendwenn brucha, weil was isch wenn die tokens vom user alle abglofa sind und er uf den Qr-Code goht? denn würd er uf die login Seite umgleitet werra und müsst den da Qr-Code nommol iscanna - und so künnt ma ean den mit dam query noch dam login no uf an spielplatz umleita

Reply:
aso du muansch quasi dass ma einfach da frontend router so ändert dass num die id in da park detail url isch sondern an kürzel/name vom platz

Reply:
ähm na net ganz - des kann ruich a id bliba - der name/kürzel vom platz söt aber als query dabei sie, so dass wenn da auth interseptor die uf login witta leitet das mir wissen wo er nochad ane will und er nocham login net einfach uf am Start-Tab isch

Reply:
aso du willsch an automatischer post login redirect sozusagen. und was hot des damit zum tuat dass du die ids weg willsch

Reply:
mh, jo mit ids künnt ma des o macha. I hätts halt einfach besser gfunda wenn ma des alles mit dam name/kürzel vom Spielplatz machen - das ma o in da url den net nur a zahl siat - und das wenn irgendwas mit da ids passiert, dass halt trotzdem die qr-codes no uf an richtiga spielplatz zoagen

Reply:
jetzt simma widda am anfang vo da diskussion. die ids im seed sin genau dafür do dass se fix sin und die qr code verlinkungen ned failen.

Reply:
jo tuat ma leid, i hob net dra denkt das da post login redirect jo o mit ids funktioniert. Mir künnen natürlich die urls mit dam Name/Kürzel jo o fix macha, es goht jo eigentlich nur no drum was uns liaba isch oder? Aber mir künnen es gern per Id macha - denn dürfa ma halt net vergeassa des Autoincrement zum entferna im Shema und des Feld "qr" weil des brüchts den jo net oder?

Reply:
stimmt qr brücht ma num. des ding isch eaba wenni die ids fix im schema setz denn wird des feld vom autoincrement ignoriert. so kama garantieren dass es imma die selben ids sin

Reply:
jo genau, des woas i. I moan bloß das wenn ma des autoincrement entfernt kann es net passiera das jemand an platz hinzufügt ohne id - da es den jo required isch die id selber zum setza

Reply:
asoo jo des künnt ma eig o maha jo